grakn-graql/src/main/java/ai/grakn/graql/internal/template/Scope.java

```diff
@@ -119,7 +118,7 @@ else if(!isGlobalScope()) {
         }
         else {
             // Unknown variable
-            return ObjectUtils.NULL;
+            return null;
```

Comment:
SO why the change to ObjectUtils.NULL? According to the java docs it is used when:

> used as a null placeholder where null has another meaning. 

Is this the case? Does null have another meaning in these contexts?